tree/dataframe/inc/ROOT/RResultPtr.hxx

```diff
@@ -170,7 +170,12 @@ public:
 
    /// Get a const reference to the encapsulated object.
    /// Triggers event loop and execution of all actions booked in the associated RLoopManager.
-   const T &GetValue() { return *Get(); }
+   const T &GetValue()
+   {
+       if( fObjPtr == nullptr)
+           throw;
```

Comment:
```suggestion
           throw std::runtime_error("Trying to access the contents of a null RResultPtr.");
```

Reply:
Yes this seems better, I thought of only terminating it for that condition so I added only `throw ;`